src/components/shared/VirtualList.js

```diff
@@ -343,31 +343,9 @@ export class VirtualList<Item> extends React.PureComponent<
   }
 
   /**
-   * Scroll the minimum amount so that the requested item is fully visible
-   * in the viewport. If the item is not already visible, this means that
-   * it'll be shown at one of the edges of the viewport.
-   * This is different from scrollToItem, which always makes the item visible
-   * at the top of the viewport.
+   * Scroll the container horizontally if necessary
    */
-  /* This method is used by users of this component. */
-  /* eslint-disable-next-line react/no-unused-class-component-methods */
-  scrollItemIntoView(itemIndex: number, offsetX: CssPixels) {
-    const container = this._container.current;
-    if (!container) {
-      return;
-    }
-    const itemTop = itemIndex * this.props.itemHeight;
-    const itemBottom = itemTop + this.props.itemHeight;
-
-    if (container.scrollTop > itemTop) {
-      container.scrollTop = itemTop;
-    } else if (container.scrollTop + container.clientHeight < itemBottom) {
-      container.scrollTop = Math.min(
-        itemTop,
-        itemBottom - container.clientHeight
-      );
-    }
-
+  _scrollContainerHorizontally(container: HTMLDivElement, offsetX: CssPixels) {
```

Comment:
Note: the offsetX we pass here is usually wrong because it doensn't take into account the fixed column widths. I intend to change this once #4204 is done.